src/polyswarm_api/__init__.py

```diff
@@ -192,62 +205,52 @@ def _fix_result(self, result):
                     logger.error('Server request failed')
                     return {'reason': 'unknown_error', 'status': 'error'}
 
-    async def post_artifact(self, artifact_data_obj, artifact_data_name, artifact_type=ArtifactType.FILE):
+    async def post_artifacts(self, artifact_data_objs, artifact_data_names, artifact_type=ArtifactType.FILE):
```

Comment:
This is theoretically a breaking change, though I don't know that anyone actually uses this directly.

I think we should either be exposing it in both the sync and async API, or not at all. My preference I think is not at all.

Reply:
Made this private.